### ISSUE TITLE
Is the timeline off for server times? 

### ISSUE THREAD

**Issue by Kenjutsuu** [MadeBaruna/pom-moe#8]
I've checked Europe and NA timelines for banners, and they seem off.

**Comment by UsaidMuhammad** [comment]
Facing similar issue.

For me, the Calyx Double drops are up but the timeline still shows 6 hours.

![image](https://github.com/MadeBaruna/pom-moe/assets/19373627/162e606e-9dfc-4df7-9dc6-3958ce81585d)

Maybe it's not properly syncing with local system time.

**Comment by Kenjutsuu** [comment]
> Facing similar issue.
> 
> For me, the Calyx Double drops are up but the timeline still shows 6 hours.
> 
> ![image](https://user-images.githubusercontent.com/19373627/252209813-162e606e-9dfc-4df7-9dc6-3958ce81585d.png)
> 
> Maybe it's not properly syncing with local system time.

Yeah, basically the time that is showing for you in the Europe server is the time that's supposed to be for the NA server. And the NA server time is completely off. 

Don't know about Asia.
